### PR TITLE
Documentation / README.md

### PR DESCRIPTION
**Pull request by dedifferentiator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,42 @@
...
+There will be `JWT-token` in response, use it wisely.
+### Create Url
+##### w/o alias
+`curl -v -X POST -H "Authorization: Bearer eyJhbGciOiJIUzUxMiJ9.eyJkYXQiOnsiZW1haWwiOiJmb28iLCJwYXNzd29yZCI6ImJhciJ9fQ.LlD2LOFalJ-mwxOBR3AqwyvObgbQ4V3A6H9bHlUFqO7vLVBxv7-uu__6r51R2AwYts71_vE2juzxWpOcCyga9g" -H "Content-Type: application/json" --data '{"url": "http://kek.lol"}' http://localhost:8080/urls/shorten`
+##### w/ alias
+`curl -v -X POST -H "Authorization: Bearer eyJhbGciOiJIUzUxMiJ9.eyJkYXQiOnsiZW1haWwiOiJmb28iLCJwYXNzd29yZCI6ImJhciJ9fQ.LlD2LOFalJ-mwxOBR3AqwyvObgbQ4V3A6H9bHlUFqO7vLVBxv7-uu__6r51R2AwYts71_vE2juzxWpOcCyga9g" -H "Content-Type: application/json" --data '{"url": "http://kek.lol", "alias": "arbidol"}' http://localhost:8080/urls/shorten`
+### Get redirected to original url
+`curl -v http://localhost:8080/r/arbidol`
+### Delete url
+`curl -v -X DELETE -H "Authorization: Bearer eyJhbGciOiJIUzUxMiJ9.eyJkYXQiOnsiZW1haWwiOiJmb28iLCJwYXNzd29yZCI6ImJhciJ9fQ.LlD2LOFalJ-mwxOBR3AqwyvObgbQ4V3A6H9bHlUFqO7vLVBxv7-uu__6r51R2AwYts71_vE2juzxWpOcCyga9g" http://localhost:8080/urls/arbidol`
```
Comment (p3rsik) [inline README.md:42]: Need to add screenshots to this `curl` requests.

Comment (dedifferentiator) [reply README.md:42]: @p3rsik ok, do it

**Review by p3rsik — APPROVED** [review]

**Comment by p3rsik** [comment]
This is awesome, senpai!11!

**Comment by dedifferentiator** [comment]
@p3rsik >~<

**Pull request merged**